test/e2e/e2e.go

```diff
@@ -206,7 +206,7 @@ var _ = ginkgo.SynchronizedAfterSuite(func() {
 // If a "report directory" is specified, one or more JUnit test reports will be
 // generated in this directory, and cluster logs will also be saved.
 // This function is called on each Ginkgo node in parallel mode.
-func RunE2ETests(t *testing.T) {
+func RunE2ETests(t *testing.T, suite string) {
```

Comment:
I need to give this a bit more thought, all this is doing is shuffling the title string around, it does really create more extensible system.  b/c if folks extend from this point it simply creates a monolith by a different name.  

Ideally I would like suite segregation so you could select by suites in a hierarchical fashion.  Other test runners have this capability, but right now it just appears that we're calling a monolith by a different name.